src/api/client/resolvers.ts

```diff
@@ -115,6 +126,29 @@ export const queryNodeStitchingResolvers = (
   },
 
   Channel: {
+    views: async (parent, args, context, info) => {
```

Comment:
There has been some improvements made to these resolvers on `dev`. We should rebase this branch and do something similar for channel views. So make sure that we don't need to fetch them on one-to-one basis but use the batched query instead. This will save us a lot of requests to Orion

Reply:
So far we only query views for individual channels for their respective views, we for sure will need this but I think it makes more sense in a different PR issue?

Reply:
If you go to `/channels` on your branch, there will be a separate Orion request made for each channel, to fetch views. We should fix this before merging

Reply:
@drillprop did some clean up on `dev` for those so should be very straightforward to introduce batched query